src/java/org/apache/cassandra/index/sai/iterators/KeyRangeIterator.java

```diff
@@ -229,14 +229,21 @@ protected static PrimaryKey nullSafeMin(PrimaryKey a, PrimaryKey b)
         if (a == null) return b;
         if (b == null) return a;
 
-        return a.compareToStrict(b) > 0 ? b : a;
+        return a.compareTo(b) > 0 ? b : a;
```

Comment:
why is this one not `b, true`?  That is the default but not following the pattern of this patch/file?